spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -34,6 +35,7 @@
       status: response['status'],
       transaction_end_date_time: response['transactionEndDateTime'],
       transaction_start_date_time: response['transactionStartDateTime'],
+      issuer: 'http://localhost:3000',
```

Comment:
Similar to what we do above with `enrollment_code`, we could check that this matches the issuer from the enrollment, which may help avoid any issues of keeping the value in sync from how it's assigned.

```suggestion
      issuer: pending_enrollment.issuer,
```